src/client/GraphicCard.cpp

```diff
@@ -0,0 +1,76 @@
+#include <client.hpp>
+#include <iostream>
+
+#define ACTION_INDEX 11
+#define PRIORITY_INDEX 7
+
+namespace client
+{
+
```

Comment:
Write a comment for each constructor to explain that one is used for priorityCards and the other for the other type

Reply:
It's done in improvement-19